cmake/ExternalGitTags.cmake

```diff
@@ -0,0 +1,239 @@
+  "URL for external ButterflyPACK build"
+)
+set(EXTERN_BUTTERFLYPACK_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external ButterflyPACK build"
+)
+set(EXTERN_BUTTERFLYPACK_GIT_TAG
+  "f40223c1c6495913302d66d9d6692f7311c76901" CACHE STRING  # 06/16/2023
+  "Git tag for external ButterflyPACK build"
+)
+
+# GKlib (for METIS and ParMETIS)
+set(EXTERN_GKLIB_URL
+  "https://github.com/KarypisLab/GKlib.git" CACHE STRING
+  "URL for external GKlib build"
+)
+set(EXTERN_GKLIB_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external GKlib build"
+)
+set(EXTERN_GKLIB_GIT_TAG
+  "8bd6bad750b2b0d90800c632cf18e8ee93ad72d7" CACHE STRING  # 03/26/2023
+  "Git tag for external GKlib build"
+)
+
+# GSLIB
+set(EXTERN_GSLIB_URL
+  "https://github.com/Nek5000/gslib.git" CACHE STRING
+  "URL for external GSLIB build"
+)
+set(EXTERN_GSLIB_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external GSLIB build"
+)
+set(EXTERN_GSLIB_GIT_TAG
+  "39d1baae8f4bfebe3ebca6a234dcc8ba1ee5edc7" CACHE STRING  # 11/09/2022
+  "Git tag for external GSLIB build"
+)
+
+# HYPRE (for MFEM)
+set(EXTERN_HYPRE_URL
+  "https://github.com/hypre-space/hypre.git" CACHE STRING
+  "URL for external HYPRE build"
+)
+set(EXTERN_HYPRE_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external HYPRE build"
+)
+set(EXTERN_HYPRE_GIT_TAG
+  "64604ec07539dc41171aad0a97976977d71d5e78" CACHE STRING  # 06/24/2023
+  "Git tag for external HYPRE build"
+)
+
+# METIS
+set(EXTERN_METIS_URL
+  "https://github.com/KarypisLab/METIS.git" CACHE STRING
+  "URL for external METIS build"
+)
+set(EXTERN_METIS_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external METIS build"
+)
+set(EXTERN_METIS_GIT_TAG
+  "e0f1b88b8efcb24ffa0ec55eabb78fbe61e58ae7" CACHE STRING  # 04/02/2023
+  "Git tag for external METIS build"
+)
+
+# MFEM
+set(EXTERN_MFEM_URL
+  "https://github.com/mfem/mfem.git" CACHE STRING
+  "URL for external MFEM build"
+)
+set(EXTERN_MFEM_GIT_BRANCH
+  "master" CACHE STRING
+  "Git branch for external MFEM build"
+)
+set(EXTERN_MFEM_GIT_TAG
+  "f3a58367375e8544b1c9e2066a9aa5b9b2211557" CACHE STRING  # master @ 06/29/2023
+  "Git tag for external MFEM build"
+)
+
+# MUMPS
+set(EXTERN_MUMPS_URL
+  "https://github.com/scivision/mumps.git" CACHE STRING
+  "URL for external MUMPS build"
+)
+set(EXTERN_MUMPS_GIT_BRANCH
+  "main" CACHE STRING
+  "Git branch for external MUMPS build"
+)
+set(EXTERN_MUMPS_GIT_TAG
+  "9ffe8ce9cc2d71798634034d4182a96426f71145" CACHE STRING  # 06/28/2023
+  "Git tag for external MUMPS build"
+)
+
+# ParMETIS
+set(EXTERN_PARMETIS_URL
+  "https://github.com/KarypisLab/ParMETIS.git" CACHE STRING
+  "URL for external ParMETIS build"
+)
+set(EXTERN_PARMETIS_GIT_BRANCH
+  "main" CACHE STRING
+  "Git branch for external ParMETIS build"
+)
+set(EXTERN_PARMETIS_GIT_TAG
+  "8ee6a372ca703836f593e3c450ca903f04be14df" CACHE STRING  # 03/26/2023
+  "Git tag for external ParMETIS build"
+)
+
+# PETSc (for SLEPc)
```

Comment:
So the future here is that we won't be using the PETSc linear algebra aside from SLEPc? So no more `PetscParVector`?

Reply:
That's right! See above.